cmd/code.go

```diff
@@ -26,10 +26,18 @@ func Code(cmd *cobra.Command, args []string) error {
 		select {
 		case e := <-execCh:
 			if e.Status == types.StatusRunning {
-				prvKey := getDefaultKey(ctx, e, prvKey)
-				ensureHosts(e, prvKey)
+				prvKey, err := getDefaultKey(ctx, e, prvKey)
+				if prvKey == "" {
+					ui.Errorf("expected private key to be none empty string")
+					os.Exit(1)
+				}
+				if err != nil {
+					ui.Errorf("failed to get private key: %s", err)
```

Comment:
The `ui.Errof` should be human consumable error (i.e. capitalized and only the info that the user needs to see).

Reply:
Great Ty will sort this